kodilla-testing2/src/main/java/com/kodilla/testing2/facebook/FacebookTestingApp.java

```diff
@@ -0,0 +1,46 @@
+package com.kodilla.testing2.facebook;
+
+import com.kodilla.testing2.config.WebDriverConfig;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class FacebookTestingApp {
+
+    public static final String XPATH_ACCEPT_COOKIE = "//button[@data-cookiebanner=\"accept_button\"]";
+    public static final String XPATH_CREATE_NEW_ACCOUNT = "//a[contains(@ajaxify, \"spotlight\")]";
+    public static final String XPATH_BIRTHDAY = "//select[@id =\"day\"]";
+    public static final String XPATH_MONTH_OF_BIRTH = "//div[contains(@class, \"_5k_5\")]/span/span/select[contains(@name, \"_month\")]";
+    public static final String XPATH_YEAR_OF_BIRTH = "//div[contains(@class, \"_5k_5\")]/span/span/select[contains(@name, \"_year\")]";
+
+    public static void main(String[] args) {
+
+        WebDriver driver = WebDriverConfig.getDriver(WebDriverConfig.CHROME);
+        driver.get("https://www.facebook.com/");
+
+        while (!driver.findElement(By.xpath(XPATH_ACCEPT_COOKIE)).isDisplayed());
+
+        WebElement cookieAcceptButton = driver.findElement(By.xpath(XPATH_ACCEPT_COOKIE));
+        cookieAcceptButton.click();
+
+        WebElement inputEmail = driver.findElement(By.xpath(XPATH_CREATE_NEW_ACCOUNT));
+        inputEmail.click();
+
+        WebElement day = new WebDriverWait(driver, 10L)
+                .until(webDriver -> webDriver.findElement(By.xpath(XPATH_BIRTHDAY)));
+
```

Comment:
tej zmiennej day z linii 31 nigdzie dalej nie używam, ale nie wiem jak to inaczej zrobić :)

Reply:
To po co ci ta zmienna? nie musisz przeciez zapamitac wyniku w zmiennej